test/mavsdk_tests/integration_test_runner/logger_helper.py

```diff
@@ -1,11 +1,10 @@
-#!/usr/bin/env python3
```

Comment:
Why did you remove this? Just curious.

Reply:
Just because it's not a script that's executed directly. The `#!` is only for the shell interpreter.

Reply:
Right, fair enough.